src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -43,72 +38,31 @@ class ScalaSettings extends Settings.SettingGroup {
   val nobootcp = BooleanSetting("-nobootcp", "Do not use the boot classpath for the scala jars.")
   val strict = BooleanSetting("-strict", "Use strict type rules, which means some formerly legal code does not typecheck anymore.")
 
-  val argfiles = BooleanSetting("@<file>", "A text file containing compiler arguments (options and source files)")
```

Comment:
Thanks for your feedback. I created this PR just for more clarification about what's to be implemented, what's to be dropped. I'll mark planned options as TODO, create issues for them and start implementing or porting.